blog/_posts/2017-02-16-anniversary-campaign.md

```diff
@@ -0,0 +1,60 @@
+
+
+Can you believe it? This is the 5th year of RGSoC! We are so happy to be reaching a quite noble age for grassroots initiatives and projects in Open Source! 5 years! 
+
+You can now celebrate with us and share the love and [donate](https://railsgirlssummerofcode.org/campaign/). 
+
+Since our beginning in 2013 we have been fighting for getting more women\* into the Open Source community by giving out scholarships that we finance through crowdfunding and  sponsorships. 
+
+We believe our mission of enabling women\* to pursue a career in tech and help to foster diversity in tech is as simple as it is radical. We believe that fostering diversity is now more important than ever and want to take a stand: We are not going anywhere. We are staying. For the next summer, the next four years - and the next five. 
+
+But we can only do it together with you.
+
+Let’s fight together for continuing our amazing success of enabling women* all over the world to dive into the world of Open Source and making the community more inclusive, diverse, and newcomer-friendly.
+
+### #5YearsofRGSoC 
+
+Today we our [anniversary funding campaign](https://railsgirlssummerofcode.org/campaign/) begins and throughout the next month we’ll celebrate our #5yearsofRGSoC with fun facts, statistics and things you didn’t know about your favorite summer scholarship program. With the campaign we hope to fund as many scholarships as we can - We’ve been able to fund more and more teams since 2013 (starting at 10, going up to 16 in the last years).
+
+### 🎂🎈🎉✊🏾💻🚀👧🏾👧🏿👧🏼👧🏻👧🏽✨🎈🎂
+
+Last year the applications for RGSoC have almost doubled (95!) and this year we expect no less! Since 2013 all we got over 600+ applications from all over the world. Thanks to our successful funding campaigns every year we could offer 145 scholarships to these wonderful women\* in the pictures above. 
+
+
+
+### The Early Bird Supports Diversity
+
+In the last few weeks, we’ve been reaching out to companies and are happy to announce that these early bird sponsors have helped to build a solid ground for our road of RGSoC 2017. Most of you will know that fundraising year after year stays extremely hard work - budgets change, companies get aquired or sold, intern goals shift or life happens. 
+
+This is why we are extremely happy to not only welcome some of our most faithful partners and sponsors (yes, you, **GitHub**, **Travis CI** and **innoQ**) again on board this year but also some news companies (OHAI **Nokia** and **Heroku**) as well as our two earliest earlybird sponsors **examsuccess, agilebloom**. 
+
+Together with the fantastic sponsors **Lauchschool, Ableton, OpenSuse, Apcera, articulate and Honeybadger** all you amazing early birds you show to us and the world that you care. About our program, our mission and diversity in tech! Thank you for that. We are so proud to be changing the world and our communities together with you step by step.
```

Comment:
“all you amazing early birds you show to us” -> “all you amazing early birds show to us” (without repeating “you”)